homeassistant/components/homekit/type_fans.py

```diff
@@ -71,20 +72,16 @@ def __init__(self, *args: Any) -> None:
 
         features = state.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
         percentage_step = state.attributes.get(ATTR_PERCENTAGE_STEP, 1)
-        self.preset_modes: list[str] | None = state.attributes.get(ATTR_PRESET_MODES)
+        self.preset_modes = state.attributes.get(ATTR_PRESET_MODES)
```

Comment:
Please keep the typing

Reply:
Oops, this must have been a mistake when merging in `upstream/dev`.